frontend/packages/console-dynamic-plugin-sdk/src/api/api.ts

```diff
@@ -8,3 +15,16 @@ const MockImpl = () => {
 
 export const useK8sWatchResource: UseK8sWatchResource = MockImpl;
 export const useK8sWatchResources: UseK8sWatchResources = MockImpl;
+
+// status, icons
+export const Status: React.FC<StatusProps> = MockImpl;
+export const GenericStatus: React.FC<GenericStatusProps> = MockImpl;
+export const ErrorStatus: React.FC<StatusComponentProps> = MockImpl;
+export const SuccessStatus: React.FC<StatusComponentProps> = MockImpl;
+export const PendingStatus: React.FC<StatusComponentProps> = MockImpl;
+export const ProgressStatus: React.FC<StatusComponentProps> = MockImpl;
+export const BlueInfoCircleIcon: React.FC<ColoredIconProps> = MockImpl;
+export const GreenCheckCircleIcon: React.FC<ColoredIconProps> = MockImpl;
+export const RedExclamationCircleIcon: React.FC<ColoredIconProps> = MockImpl;
+export const YellowExclamationTriangleIcon: React.FC<ColoredIconProps> = MockImpl;
```

Comment:
I prefer to name these icons with their functions rather than their visuals since they fulfill specific use cases.

eg. InfoIcon, SuccessIcon, ErrorIcon, WarningIcon

Reply:
fixed @christianvogt